pimd/pim6_cmd.c

```diff
@@ -625,6 +625,102 @@ DEFPY (interface_no_ipv6_mld_query_interval,
 				    "frr-routing:ipv6");
 }
 
+DEFPY (mld_group_watermark,
```

Comment:
Inconsistent naming - the other commands are named `ipv6_mld_something`, this one `mld_something` - please add the `ipv6_` in front